plugin/proto/plugin.proto

```
@@ -0,0 +1,176 @@
+syntax = "proto3";
+
+package missioncontrol.plugin.v1;
```

Buf (1.69.0)

[error] Files with package "missioncontrol.plugin.v1" must be within a directory "missioncontrol/plugin/v1" relative to root but were in directory "plugin/proto".

(PACKAGE_DIRECTORY_MATCH)